DNN Platform/DotNetNuke.DependencyInjection/Extensions/TypeExtensions.cs

```diff
@@ -26,10 +23,13 @@ public static class TypeExtensions
         /// <param name="assembly">
         /// The assembly to retrieve all types from.
         /// </param>
+        /// <param name="logger">
+        /// A optional <see cref="ILog"/> object. This will log any messages from the exception catching to the logs as an Error.
+        /// </param>
         /// <returns>
         /// An array of all <see cref="Type"/> in the given <see cref="Assembly"/>.
-        /// </returns>
-        public static Type[] SafeGetTypes(this Assembly assembly)
```

Comment:
This is a breaking change, can we use an overload instead of optional parameters?

Reply:
Yes, I will add an overload and remove the optional parameter.

Reply:
OK PR updated